src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,33 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int cargoCapacity;
```

Comment:
Don't forget about access modifiers for your custom fields in `Truck` class. It's a good practice to make fields private and provide public getters and setters to access these fields.